README.md

```diff
@@ -78,7 +82,7 @@ From the root of the repository, install the tools used to develop.
 
 Build the font:
 
-    $:/src/icons make
+    $ make -C src/icons
```

Comment:
Technically the make command was not wrong.

`$:/src/icons` denotes the directory. I agree, it looks a bit strange and probably should rather look like `[src/icons]$ make`.
But yes, maybe your command is cleaner.

Reply:
I have never seen this syntax before, the `:` at the beginning. So I even tried to type it in the terminal to see if it worked :)

In any case, `make -C` is a common way to invoke make in a subdir. `cd src/icons && make && cd -` also works, but it's less concise and elegant.

Reply:
Right, I know that. I'm just saying that the prompt (even if it looks weird) is not a wrong statement.
But ok, your make -C looks cleaner.